clients/src/main/java/org/apache/kafka/common/protocol/ApiKeys.java

```diff
@@ -108,6 +108,8 @@
     UNREGISTER_BROKER(ApiMessageType.UNREGISTER_BROKER, false, RecordBatch.MAGIC_VALUE_V0, true),
     DESCRIBE_TRANSACTIONS(ApiMessageType.DESCRIBE_TRANSACTIONS),
     LIST_TRANSACTIONS(ApiMessageType.LIST_TRANSACTIONS),
+    GET_TELEMETRY_SUBSCRIPTION(ApiMessageType.GET_TELEMETRY_SUBSCRIPTION, true, true),
```

Comment:
Somewhat nit-picky: the KIP lists _subscription_ in the plural: _subscriptions_. Not sure if we care, but it is different.

Reply:
Aight, most kafka APIs are plural. ListOffsets, InitTransactions, CreateTopics, etc.